setup.py

```diff
@@ -72,6 +72,8 @@ def get_version(filename):
         'jsonschema~=3.0',
         'pathlib2>=2.3.0;python_version<="3.4"',
         'importlib_resources~=1.0.2;python_version<"3.7"',
+        'networkx~=2.4;python_version>="3.5"',
+        'networkx~=2.1;python_version<"3.5"'
```

Comment:
Will need to vet new dependencies

Reply:
In terms of [license](https://github.com/networkx/networkx/blob/master/LICENSE.txt) we should be ok: https://tldrlegal.com/license/bsd-3-clause-license-(revised)